framework/di/Container.php

```diff
@@ -426,7 +426,9 @@ protected function getDependencies($class)
         $constructor = $reflection->getConstructor();
         if ($constructor !== null) {
             foreach ($constructor->getParameters() as $param) {
-                if ($param->isDefaultValueAvailable()) {
+                if ($param->isVariadic()) {
```

Comment:
Method `isVariadic()` is available since PHP 5.6. This change will lead to fatal error on PHP 5.4 and 5.5